.gitignore

```diff
@@ -19,3 +19,4 @@ tags
 # End of https://www.gitignore.io/api/vim
 
 compile_commands.json
+.vscode
```

Comment:
This is very developer specific. Instead of adding it he, you can add it in `.git/info/exclude`. That file works like `.gitignore` but just for your local repository, so it isn't pushed